tests/nodenext/tsconfig.json

```diff
@@ -0,0 +1,12 @@
+{
+  "compilerOptions": {
+    "module": "NodeNext",
+    "target": "ES2022",
+    "outDir": "./dist",
+
```

Comment:
nit: remove empty line and order the options alphabetically.

Reply:
done!